api/src/main/java/marquez/service/models/LineageEvent.java

```diff
@@ -533,8 +533,8 @@ public static class ColumnLineageOutputColumn extends BaseJsonModel {
   @ToString
   public static class ColumnLineageInputField extends BaseJsonModel {
 
-    @NotNull private String datasetNamespace;
```

Comment:
Fix to previous PR to align with Openlineage spec: 
https://github.com/OpenLineage/OpenLineage/blob/main/spec/facets/ColumnLineageDatasetFacet.json